README.md

```diff
@@ -55,26 +55,29 @@ The easiest way to compile the code and run the system locally is using [Docker]
 * [Install Docker](https://docs.docker.com/get-docker/)
 * [Install docker-compose](https://docs.docker.com/compose/install/)
 
-Don't forget to run the docker daemon!
+### Recommended post-install steps
+- Enable the Docker daemon as detailed [here](https://docs.docker.com/engine/install/linux-postinstall/#configure-docker-to-start-on-boot)
+  - It is recommended to read the other instructions here to customize other features you may find useful
+- Follow [these instructions](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) to enable the ability to run docker as a non-root user
```

Comment:
I'm trying to figure out if there's a sensible way to rephrase this to also have a more descriptive link text, but nothing's immediately coming to mind for me…

Reply:
changed it a bit